core/templates/dev/head/galleries/Gallery.js

```diff
@@ -275,6 +284,7 @@ oppia.controller('SearchBar', [
       summary: ''
     }
   };
+  $scope.translationData = {}
```

Comment:
It's a bit unclear what this variable represents. Can you give it a name that represents what it is, rather than what it's used for?